docs/_shared/Example.tsx

```diff
@@ -93,7 +93,7 @@ function Example({ source, testId, paddingBottom }: ExampleProps) {
   );
 
   return (
-    <>
+    <React.Fragment>
```

Comment:
What the motivation?

Reply:
Why are we using `<React.Framgent>` in the codebase? I think that it's between these lines:

- This approach allows setting a key 
- This approach allows new developers googling what the element for.

If you think that we should change the tradeoff, please open an RFC on the main repository, the objective here is to share the same eslint configuration on the whole codebase.